src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```
@@ -388,6 +389,13 @@ LANGUAGE_COOKIE: {{ env "ENVIRONMENT" }}-openedx-language-preference
 MIT_LEARN_AI_API_URL: https://{{ key "edxapp/learn-api-domain" }}/ai  # Added for ol_openedx_chat
 MIT_LEARN_API_BASE_URL: https://{{ key "edxapp/learn-api-domain" }}/learn  # Added for ol_openedx_chat
 MIT_LEARN_SUMMARY_FLASHCARD_URL: https://{{ key "edxapp/learn-api-domain" }}/learn/api/v1/contentfiles/  # Added for ol_openedx_chat
+MIT_LEARN_BASE_URL: "https://{{ key edxapp/learn-frontend-domain}}"
+MIT_LEARN_SUPPORT_EMAIL: "mitlearn-support@mit.edu"
+UAI_COURSE_KEY_FORMATS:
+  - course-v1:uai_source
+  - course-v1:uai_hhc
+  - course-v1:uai_mit
```

Not sure if this makes a difference but these are all caps, and there is one more. 


```suggestion
  - course-v1:UAI_SOURCE
  - course-v1:UAI_HHC
  - course-v1:UAI_MIT
  - course-v1:UAI_ET
```

---

Can it match on just `UAI_`?

---

Path matching is case-insensitive. We normalize it by converting to lowercase first.

> Can it match on just UAI_?

Yes, I can adjust the condition to look for course-v1:UAI_ within the URL path.